src/entity/aggregator/new_aggregator.rs

```diff
@@ -43,6 +44,11 @@ fn https(url: &str) -> Result<(), ValidationError> {
     Ok(())
 }
 
+#[cfg(feature = "integration-testing")]
+fn https(_: &str) -> Result<(), ValidationError> {
```

Comment:
I somewhat prefer this to be an environment variable check instead, i.e. in the original `https()` function check if something like `ALLOW_INSECURE_HTTP` is set. This is in the interest of keeping the behavior between integration-testing builds and release builds as close as possible.

I don't feel strongly about this, however. Feel free to disregard.